test/rules/verify-user-email-with-password-reset.test.js

```diff
@@ -73,4 +73,20 @@ describe(ruleName, () => {
       done();
     });
   });
+
+  it('should do nothing if patch request fails', (done) => {
```

Comment:
let's add another test for non-200 response (corresponding to other comment for `verify-user-email-with-password-reset.js`)